src/adapter/src/config/frontend.rs

```diff
@@ -11,6 +11,7 @@ use std::{collections::BTreeMap, time::Duration};
 
 use derivative::Derivative;
 use launchdarkly_server_sdk as ld;
+use mz_sql::catalog::EnvironmentId;
```

Comment:
`mz_` imports should go into a separate import block after third-party imports.

Meta: I think this is *the* most common style nit I have on PRs. The issue is that rust-analyzer does not know to treat mz imports specially and we have nothing in CI that checks for this. Maybe I can come up with something during a skunkworks. Maybe we could even configure rustfmt to do this check for us.

Reply:
https://github.com/MaterializeInc/materialize/pull/16424

Reply:
Yeah, we could definitely write some shell script. And I think we need to actually. rustfmt has [a relevant (unstable) option](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#group_imports), but it doesn't support configuring the grouping we want.